### ISSUE TITLE
W3C Validator errors

### ISSUE THREAD

**Issue by Tamayuzu** [ableplayer/ableplayer#432]
Hey guys!

We sent a sample using Able Player to our department's accessibility testers and they came back with the following issue:

-------------------------------------------

TEST CASE
**Details:** Ran the code for the video player through W3C Validator 
**Result:** 2 errors. 

**Error: Attribute orientation not allowed on element div at this point.** 
From line 47, column 937; to line 47, column 1157 
`<div orientation="horizontal" class="able-seekbar-head" tabindex="0" role="slider" aria-label="video…min="0" aria-valuemax="4788.096" style="left: -6.4px;" aria-valuetext="0 seconds" aria-valuenow="0">`

**Error: Element li is missing required attribute aria-checked.** 
From line 47, column 13306; to line 47, column 13344 
`<li role="menuitemradio" tabindex="-1">`

I'm thinking we can flag these as false positives but any insight into these errors would be helpful!

Cheers'

**Comment by terrill** [comment]
Thanks for reporting these errors @Tamayuzu!  

They were legitimate errors: 

- `orientation="horizontal"` should have been `aria-orientation="horizontal"`  
- the `aria-checked` attribute was missing from the Captions Off menu item if that item was not selected (should have been `aria-checked="false"`. 

These are both fixed as of cd6b8a7e43f6d97bd2f7260e4f37bfab85ed1454
